gprofiler/cloud_metadata.py

```diff
@@ -0,0 +1,161 @@
+from dataclasses import dataclass
```

Comment:
Add file header (copy from another one)

Reply:
Dataclasses completely failed on me on Python 3.6 in mypy, so I gave up on them in #134